src/zmalloc.c

```diff
@@ -181,6 +181,14 @@ void *ztrycalloc_usable(size_t size, size_t *usable) {
 #endif
 }
 
+/* Allocate memory and zero it or panic.
+ * We need this wrapper to have a calloc compatible signature */
+void *zcalloc_num(size_t num, size_t size) {
+    void *ptr = ztrycalloc_usable(num*size, NULL);
```

Comment:
maybe put an overflow check here too?

Reply:
being discussed in
https://github.com/redis/redis/pull/9462/files/90523e04ddb46b7a35776c5376ddd73d5dcea6a2..2b66f5249f4ad3623bde77def23de6b9f79706ed#r778617183